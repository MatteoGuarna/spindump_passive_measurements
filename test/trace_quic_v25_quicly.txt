A test with the Quicly QUIC implementation server and Quant client.

There are six sessions:

- regular + full start
- regular + 0rtt
- regular + larger file
- regular  + h3
- key update
- quantum

The commands used were:

  (rm /tmp/quant-session;
  ./Bin/client -i en0 https://quic.examp1e.net:4433; 
  ./Bin/client -i en0 https://quic.examp1e.net:4433; 
  ./Bin/client -i en0 https://quic.examp1e.net:4433/30000;
  ./Bin/client -3 -i en0 https://quic.examp1e.net:443; 
  ./Bin/client -u -i en0 https://quic.examp1e.net:4433; 
  ./Bin/client -m -i en0 https://quic.examp1e.net:4433)
  
  
  
