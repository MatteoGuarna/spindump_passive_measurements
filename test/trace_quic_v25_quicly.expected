QUIC 100.72.46.172 <-> 133.242.206.244 6d8c566ea82ca2-4a911dcd (61908:4433) at 1580909591364448 new starting packets 0 0 bytes 0 0 note "No rsp,v.0xbabababa,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 6d8c566ea82ca2-ae0125e8144a91 (61908:4433) at 1580909591627731 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 6d8c566ea82ca2-ae0125e8144a91 (61908:4433) at 1580909591627731 measurement starting right 263283 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 null-ae0125e8144a91 (61908:4433) at 1580909591629845 measurement starting left 2114 packets 1 1 bytes 1228 63 bandwidth 1228 63 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 null-086c258f0a5e9500 (61908:4433) at 1580909591921712 change starting packets 2 1 bytes 2456 63 bandwidth 2456 63 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 null-086c258f0a5e9500 (61908:4433) at 1580909591921712 measurement up right 291867 packets 2 1 bytes 2456 63 bandwidth 2456 63 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 57e6d6ae-086c258f0a5e9500 (61908:4433) at 1580909591921994 packet up responder length 1308 packets 2 3 bytes 2456 2679 bandwidth 2456 2679 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 57e6d6ae-086c258f0a5e9500 (61908:4433) at 1580909591922271 packet up responder length 906 packets 2 4 bytes 2456 3585 bandwidth 2456 3585 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 57e6d6ae-086c258f0a5e9500 (61908:4433) at 1580909591930237 packet up initiator length 72 packets 3 4 bytes 2528 3585 bandwidth 2528 3585 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 57e6d6ae-086c258f0a5e9500 (61908:4433) at 1580909591930487 packet up initiator length 71 packets 4 4 bytes 2599 3585 bandwidth 2599 3585 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 57e6d6ae-086c258f0a5e9500 (61908:4433) at 1580909591932127 packet up initiator length 1500 packets 5 4 bytes 4099 3585 bandwidth 4099 3585 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 57e6d6ae-086c258f0a5e9500 (61908:4433) at 1580909592100199 packet up responder length 1303 packets 5 5 bytes 4099 4888 bandwidth 4099 4888 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 04839986-e8a8568dd20361c899e45957acac81da04 (60749:4433) at 1580909593722773 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 04839986-08c873ef0a0759c1 (60749:4433) at 1580909594071587 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 04839986-08c873ef0a0759c1 (60749:4433) at 1580909594071587 measurement up right 348814 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 83998672-08c873ef0a0759c1 (60749:4433) at 1580909594073638 packet up initiator length 144 packets 2 1 bytes 1372 580 bandwidth 1372 580 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 83998672-08c873ef0a0759c1 (60749:4433) at 1580909594073937 packet up initiator length 1500 packets 3 1 bytes 2872 580 bandwidth 2872 580 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 83998672-08c873ef0a0759c1 (60749:4433) at 1580909594187184 packet up responder length 452 packets 4 2 bytes 2929 1032 bandwidth 2929 1032 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 04121055-a5a4b1ceab4b4f27d315f2f4785fff37368326 (53621:4433) at 1580909595813929 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 04121055-08b6e47193cb5f3f (53621:4433) at 1580909596080669 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 04121055-08b6e47193cb5f3f (53621:4433) at 1580909596080669 measurement up right 266740 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 121055c2-08b6e47193cb5f3f (53621:4433) at 1580909596083624 packet up initiator length 144 packets 2 3 bytes 1372 3924 bandwidth 1372 3924 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 121055c2-08b6e47193cb5f3f (53621:4433) at 1580909596084264 packet up initiator length 1500 packets 3 3 bytes 2872 3924 bandwidth 2872 3924 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 c90ffaefd81f8d45eb041030-null (50579:443) at 1580909598058810 new starting packets 0 0 bytes 0 0 note "No rsp,v.0xbabababa,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 c90ffaefd81f8d45eb041030-1030ae1f09c90f (50579:443) at 1580909598372906 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 c90ffaefd81f8d45eb041030-1030ae1f09c90f (50579:443) at 1580909598372906 measurement starting right 314096 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 null-1030ae1f09c90f (50579:443) at 1580909598375075 measurement starting left 2169 packets 1 1 bytes 1228 52 bandwidth 1228 52 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 null-09fbc7e308ef2adca7 (50579:443) at 1580909598679412 change starting packets 2 1 bytes 2456 52 bandwidth 2456 52 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 null-09fbc7e308ef2adca7 (50579:443) at 1580909598679412 measurement up right 304337 packets 2 1 bytes 2456 52 bandwidth 2456 52 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 6a9ed247-09fbc7e308ef2adca7 (50579:443) at 1580909598679418 packet up responder length 1308 packets 2 3 bytes 2456 2668 bandwidth 2456 2668 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 6a9ed247-09fbc7e308ef2adca7 (50579:443) at 1580909598679759 packet up responder length 1079 packets 2 4 bytes 2456 3747 bandwidth 2456 3747 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 6a9ed247-09fbc7e308ef2adca7 (50579:443) at 1580909598681670 packet up initiator length 73 packets 3 4 bytes 2529 3747 bandwidth 2529 3747 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 6a9ed247-09fbc7e308ef2adca7 (50579:443) at 1580909598681883 packet up initiator length 72 packets 4 4 bytes 2601 3747 bandwidth 2601 3747 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 6a9ed247-09fbc7e308ef2adca7 (50579:443) at 1580909598683857 packet up initiator length 1500 packets 5 4 bytes 4101 3747 bandwidth 4101 3747 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 6a9ed247-09fbc7e308ef2adca7 (50579:443) at 1580909598986608 packet up responder length 1080 packets 6 5 bytes 4196 4827 bandwidth 4196 4827 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 6a9ed247-09fbc7e308ef2adca7 (50579:443) at 1580909598987877 packet up initiator length 74 packets 7 8 bytes 4270 6162 bandwidth 4270 6162 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 0431b891-450b7a20586ceee0c3d03ecc0a340f27f124190f (49154:4433) at 1580909600541393 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 0431b891-0873c71f85b26087 (49154:4433) at 1580909600807747 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 0431b891-0873c71f85b26087 (49154:4433) at 1580909600807747 measurement up right 266354 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 31b8915e-0873c71f85b26087 (49154:4433) at 1580909600811056 packet up initiator length 144 packets 2 1 bytes 1372 580 bandwidth 1372 580 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 31b8915e-0873c71f85b26087 (49154:4433) at 1580909600811537 packet up initiator length 1500 packets 3 1 bytes 2872 580 bandwidth 2872 580 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 31b8915e-0873c71f85b26087 (49154:4433) at 1580909601004943 packet up responder length 452 packets 4 2 bytes 2929 1032 bandwidth 2929 1032 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 04e6993d-ad4c76db2befc34ab5ba89947ce061e05f1428 (53695:4433) at 1580909602192363 new starting packets 0 0 bytes 0 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 04e6993d-ad4c76db2befc34ab5ba89947ce061e05f1428 (53695:4433) at 1580909602192364 packet starting initiator length 612 packets 2 0 bytes 1892 0 bandwidth 1892 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 04e6993d-08dee82cbd41a8cd (53695:4433) at 1580909602467618 change starting packets 2 0 bytes 1892 0 bandwidth 1892 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 04e6993d-08dee82cbd41a8cd (53695:4433) at 1580909602467618 measurement up right 275254 packets 2 0 bytes 1892 0 bandwidth 1892 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 e6993d90-08dee82cbd41a8cd (53695:4433) at 1580909602467626 packet up responder length 580 packets 2 2 bytes 1892 653 bandwidth 1892 653 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 e6993d90-08dee82cbd41a8cd (53695:4433) at 1580909602470983 packet up initiator length 144 packets 3 2 bytes 2036 653 bandwidth 2036 653 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 e6993d90-08dee82cbd41a8cd (53695:4433) at 1580909602471383 packet up initiator length 1500 packets 4 2 bytes 3536 653 bandwidth 3536 653 note "V25,no spin"
QUIC 100.72.46.172 <-> 133.242.206.244 e6993d90-08dee82cbd41a8cd (53695:4433) at 1580909602658571 packet up responder length 452 packets 5 3 bytes 3593 1105 bandwidth 3593 1105 note "V25,no spin"
